src/components/contacts/ContactItem.vue

```diff
@@ -67,6 +65,12 @@
   </div>
 </template>
 
+<style>
+  .active {
+    background-color: rgb(221,221,221);
```

Comment:
This should use one of the css variables likely so that it works in darkmode and lightmode.

Reply:
Also, you can use scss variables here, but I have had problems getting the scss variables working in the general css files, so the colors are duplicated :/